packages/node_modules/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -467,9 +468,9 @@ describe('plugin-meetings', () => {
           meeting.meetingFSM.join();
           meeting.meetingRequest.leaveMeeting = sinon.stub().returns(Promise.resolve({body: 'test'}));
           meeting.locusInfo.onFullLocus = sinon.stub().returns(true);
```

Comment:
Do these need changed to Promises also? If not, please comment as to why the 3 need it.

Reply:
I'll look